utils/dumpserver/server.py

```diff
@@ -28,4 +28,4 @@ def post_dump():
     with open(metadata_filepath, 'w') as f:
         f.write(json.dumps(metadata))
 
-    return 'ok'
+    return '', 204
```

Comment:
Not sure if this should be `''` (empty string/empty body) or something like `None` (no body at all). Maybe it doesn't matter in this particular python library or maybe it doesn't make the distinction, but I could imagine there being a conceptual difference 😇.

Reply:
using `''` seems to be the general 'way' of returning 204 for flask... or at least that's quick google / stack overflow thinks :smile: